modules/jenkins/example/configuration/new_project/jenkins_project.tfvars

```diff
@@ -0,0 +1,23 @@
+jenkins_projects = {
+
+  test_job  = {
+    # folderName          = "test_folder7"
```

Comment:
Can we remove the commented out folder configs since we are descoping it?

Reply:
We can, but I wanted to keep this code just in case when we or someone will want to extend this functionality.

Reply:
Makes sense.